### ISSUE TITLE
[button] icon button missing accessible name

### ISSUE THREAD

**Issue by yinonov** [Vonage/vivid-3#132]
icon button should auto define aria-label for the button control (unless is already defined by author).
Failing to do so result in _[Buttons do not have an accessible name](https://web.dev/button-name/?utm_source=lighthouse&utm_medium=devtools)_

**Comment by yinonov** [comment]
Still need to attend
